content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
+package_version: 2.0
+authors:
+  - Richèl J.C. Bilderbeek
+date: 2019-01-06
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+  - R
+  - community
+  - software
+  - packages
+  - phylogeny
+  - phylogenetics
+  - BEAST
+  - BEAST2
+  - Bayesian
+---
+
+`babette` [^1] is a package to work with BEAST2 [^2], 
```

Comment:
Remove backticks from all package names. Our formatting rules have changed in the last year. For formatting of package names, functions, and code, follow the [tidyverse style guidance](https://style.tidyverse.org/documentation.html#r-code).

Reply:
Done :+1: